okapi-core/src/main/java/org/folio/okapi/managers/ProxyService.java

```diff
@@ -772,7 +772,7 @@ private static void dumpHeaders(MultiMap headers) {
     for (String name : headers.names()) {
       List<String> values = headers.getAll(name);
       for (String value : values) {
-        logger.info("{}: {}", name, value);
+        logger.debug("{}: {}", name, value);
       }
```

Comment:
We have this info dump because there was a problem with headers being too large (HTTP violation). Just to tell you why this was done..  We could set it to debug, ok. Especially since it's not a problem anymore.